profiles/lib/profile_v2.lua

```diff
@@ -0,0 +1,9 @@
+-- profile api version 2
```

Comment:
just a refactor to make it simpler to create a profile. all the profiles include almost the same files- can be revered if you like.